HelloWorld.java

```diff
@@ -4,5 +4,6 @@ public class HelloWorld {
     public static void main(String[] args) {
         System.out.println("Hello, World!");
     }
+    System.put.println("こんにちは世界！")

```

Comment:
下記のようにしないと`こんにちは世界!`が表示されないはずです。
```suggestion
        System.out.println("Hello, World!");
        System.put.println("こんにちは世界！");
    }
```